pkg/webhooks/networkpolicies/networkpolicies.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	hookconfig "github.com/openshift/managed-cluster-validating-webhooks/pkg/config"
 	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+	k8sapi "k8s.io/api/networking/v1"
```

Comment:
```suggestion
	networkingv1 "k8s.io/api/networking/v1"
```
Definitely a nit, but there are a lot of k8s api imports, this would align with the other imports below

Reply:
oh right, thanks for pointing that out